src/main/java/organice/model/person/Tissuetype.java

```diff
@@ -0,0 +1,81 @@
+package organice.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static organice.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's tissue type in ORGANice.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTissuetype(String)}
+ */
+public class Tissuetype {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Tissue types are 6 integers from 1 to 12 separated by commas"
+                + " An example will be tt/1,2,3,4,5,6";
+    public final String value;
+
+    /**
+     * Constructs a {@code Tissuetype}.
+     *
+     * @param tissuetype A valid tissue type
+     */
+    public Tissuetype (String tissuetype) {
+        requireNonNull(tissuetype);
+        checkArgument(isValidTissuetype(tissuetype), MESSAGE_CONSTRAINTS);
+        value = tissuetype;
+    }
+
+    /**
+     * Returns true if a given string is a valid tissue type.
+     */
+    public static boolean isValidTissuetype(String test) {
+        String[] tissuetypeValue = test.split(",");
+        if (tissuetypeValue.length != 6 || isDuplicated(tissuetypeValue)) {
+            return false;
+        } else {
```

Comment:
Can we drop the else block to reduce indentation by 1 level? What does the code style say about this?